tortoisebeacon/tortoise_beacon.go

```diff
@@ -584,15 +555,17 @@ func (tb *TortoiseBeacon) proposalPhaseImpl(ctx context.Context, epoch types.Epo
 		tb.validProposals[epoch] = make(map[proposal]struct{})
 	}
 
-	tb.validProposals[epoch][util.Bytes2Hex(proposedSignature)] = struct{}{}
+	// TODO(nkryuchkov): encode as string instead of hex
```

Comment:
outdated TODO?

Reply:
Yes, it doesn't exist anymore in new commits. but they are not pushed yet